docker/compose.dev.yml

```diff
@@ -9,6 +9,7 @@ services:
       - "--entryPoints.web.address=:80"
       - "--entrypoints.web.forwardedHeaders.insecure=true"
       - "--entrypoints.web.proxyProtocol.insecure=true"
+      - "--log.level=DEBUG"
```

Comment:
@prasadtalasila I could create a blank directory `traefik-logs` and mount it to the container, and redirect General Logs and [Access Logs](https://doc.traefik.io/traefik/observability/access-logs/#access-logs) into it as well.

Should I go ahead with this? If so, where should I create the directory?

Reply:
Let us skip this for now.